src/Microsoft.Diagnostics.Monitoring.WebApi/Models/CapturedParametersResults.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Models
+{
+    public class CapturedParameter
+    {
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+
+        [JsonPropertyName("type")]
+        public string Type { get; set; }
+
+        [JsonPropertyName("module")]
+        public string TypeModuleName { get; set; }
+
+        [JsonPropertyName("value")]
+        public string Value { get; set; }
+
+        [JsonPropertyName("isIn")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
+        public bool IsInParameter { get; set; }
```

Comment:
What's the usage of these three `In*` properties? While these cover most of the common modifiers, there are others such as `params`, `ref readonly`, and `scoped` that are not captured. There's also nullability and default values too.

I'm wondering if these should be left off (for JSON serialization, I'm fine with the text serialization) until they can be fully captured and uniformly described.